build.sbt

```diff
@@ -74,7 +74,8 @@ lazy val client = project
       "dev.zio"                       %% "zio-test-magnolia"             % zioV % "it,test",
       "com.softwaremill.sttp.client3" %% "core"                          % sttpV,
       "com.softwaremill.sttp.client3" %% "circe"                         % sttpV,
-      "com.softwaremill.sttp.client3" %% "async-http-client-backend-zio" % sttpV
+      "com.softwaremill.sttp.client3" %% "async-http-client-backend-zio" % sttpV,
+       "com.softwaremill.sttp.client3" %% "httpclient-backend-zio"       % sttpV % "it,test"
```

Comment:
Don't need the new dependency right? The `stubLayer` should be available from `AsyncHttpClientBackend` too.

Reply:
Yes, but I was not able to get that to compile. This is the different
```
// Error Channel is Throwable (in async-http-client-backend-zio)
val stubLayer: Layer[Throwable, SttpClient with SttpClientStubbing] =
    SttpClientStubbing.layer

// Error channel is Nothing (in httpclient-backend-zio)
val stubLayer: ZLayer[Any, Nothing, SttpClientStubbing with SttpClient] = SttpClientStubbing.layer
```
And because of the covariant nature of the error channel type parameter, you could not assign a `Throwable` to `Nothing`. 

However, I pushed a new change. I have done a small hack to do the conversion.